SignUpValidation/function.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I don't think this functionality should be implemented as an Azure function. Why not making it part of the API?

Reply:
Yep is a good idea, I created as azure function because the sign-up validation was part of the azure scope and azure provides some examples with azure function to validate the sign-up process with the `API Connector` tool, but I can create a ticket to migrate this function to the API, because now is implemented in the user flow.